core/water_quality.mcf

```diff
@@ -0,0 +1,221 @@
+## Top-level Properties
+
+Node: dcid:waterQualityIndex
+typeOf: schema:Property
+domainIncludes: dcs:BodyOfWater
+rangeIncludes: schema:Number
+name: "waterQualityIndex"
+
+Node: dcid:waterLocation
+typeOf: schema:Property
+domainIncludes: dcs:BodyOfWater
+rangeIncludes: dcs:WaterLocationEnum
+name: "waterLocation"
+
+Node: dcid:waterProperty
+typeOf: schema:Property
+domainIncludes: dcs:BodyOfWater
+rangeIncludes: dcs:WaterChemicalPropertyEnum, dcs:WaterPollutantEnum
+name: "waterProperty"
+
+
+## Enum classes
+
+Node: dcid:WaterLocationEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "WaterLocationEnum"
+description: "Enumeration of water bodies" 
+
+Node: dcid:WaterPollutantEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "WaterPollutantEnum"
+description: "Enumeration of water-specific pollutants and chemical agents"
+
+Node: dcid:WaterChemicalPropertyEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "WaterChemicalPropertyEnum"
+description: "Enumeration of chemical properties of water"
+
+
+## Water Location Enum Instances
+
+Node: dcid:GroundWater
+typeOf: dcs:WaterLocationEnum
+name: "Ground Water"
+
+Node: dcid:SurfaceWater
+typeOf: dcs:WaterLocationEnum
+name: "Surface Water"
+
+
+## Water Properties Classification (Pollutant or Chemical Property)
+
+Node: dcid:waterPollutant
+typeOf: schema:Property
+rangeIncludes: dcs:WaterPollutantEnum
+name: "waterPollutant"
+description: "Water-specific Pollutants"
+
+Node: dcid:waterChemicalProperty
```

Comment:
Can we reuse waterProperty itself defined above that has rangeIncludes: dcs:WaterChemicalPropertyEnum?